lang/sql.md

```diff
@@ -0,0 +1,120 @@
+
+Note: Web SQL Database allows you to create SQL databases and make SQL calls on the client side. It is based on the popular and open source SQLite engine. Web SQL is supported by Chrome, Opera and Safari browsers.
+
+**What is Relational Database**
+
+A relational database is a database divided into logical units called tables, where tables are related to one another within the database. Relational database allows data to be broken down into logical, smaller, and manageable units for easier maintenance and better performance.
+
+Tables are related to one another through common keys or fields in a relational database system, that's why even though the desired data may exist in more than one table, you can easily join multiple tables together to get combined data set using a single query.
+
+Note: SQL became a standard of the American National Standards Institute (ANSI) in 1986, and of the International Organization for Standardization (ISO) in 1987. Although, most SQL code is not completely portable among different database systems without adjustments.
+
+**Setting Up Work Environment for Practicing SQL**
+
+If you have no DBMS (Database Management System) already installed on your system, you have several options to choose from. You can install a free, open-source DBMS.
+
+MySQL is the most popular and widely supported open-source database management system. It is very easy to download and use and available for both Windows and Linux (or UNIX) operating system. You can download it freely from here https://dev.mysql.com/downloads/mysql/
+
+You can also install SQL Server Express. It is a free version of Microsoft SQL Server which allows up to 10GB of database storage. You can download the express edition from here https://www.microsoft.com/en-in/download/details.aspx?id=30438.
+
+Alternatively, if you're planning to develop an application with PHP and MySQL you can install WampServer or XAMPP. WampServer is a Windows web development environment. It allows you to create web applications with Apache2, PHP and a MySQL database. It will also provide the MySQL administrative tool PhpMyAdmin to easily manage your databases using a web browser. You can download it from here http://www.wampserver.com/en/
+
+**⚙️ Which Version Should I Use?**
+
+Long story short, it’s entirely up to you which version of SQL you use! There’s no right or wrong answer, and it’s painless to make the jump from one version to another should you realize you need to do so later on.
+
+## 💻 Popular SQL IDEs/Editors
+
+- **Adminer**
+- **DBeaver**
+- **HeidiSQL**
+- **RazorSQL**
+- **Valentina Studio**
+
+## 🎉 Some cool projects in SQL
+
+- **. Online Phone Shop Display**
+- **e-ticket Booking**
+- **Book Store Inventory Management**
+- **Library Management System**
+- **Bank Database Display** 
```

Comment:
Might want to provide links to projects that have been made using SQL. Let us know if you need any help. 😄